pkg/languagedetection/detector_nix_test.go

```diff
@@ -138,29 +162,95 @@ func TestGetExe(t *testing.T) {
 }
 
 func BenchmarkDetectLanguage(b *testing.B) {
-	commands := [][]string{
-		{"Python", "--version"},
-		{"python3", "--version"},
-		{"py", "--version"},
-		{"Python", "-c", "import platform; print(platform.python_version())"},
-		{"python3", "-c", "import platform; print(platform.python_version())"},
-		{"py", "-c", "import platform; print(platform.python_version())"},
-		{"Python", "-c", "import sys; print(sys.version)"},
-		{"python3", "-c", "import sys; print(sys.version)"},
-		{"py", "-c", "import sys; print(sys.version)"},
-		{"Python", "-c", "print('Python')"},
-		{"python3", "-c", "print('Python')"},
-		{"py", "-c", "print('Python')"},
-		{"Java", "-version"},
-		{"Java", "-jar", "myapp.jar"},
-		{"Java", "-cp", ".", "MyClass"},
-		{"javac", "MyClass.Java"},
-		{"javap", "-c", "MyClass"},
+	commands := []struct {
+		cmdline []string
+		comm    string
+	}{
+		{
+			cmdline: []string{"Python", "--version"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"python3", "--version"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"py", "--version"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"Python", "-c", "import platform; print(platform.python_version())"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"python3", "-c", "import platform; print(platform.python_version())"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"py", "-c", "import platform; print(platform.python_version())"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"Python", "-c", "import sys; print(sys.version)"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"python3", "-c", "import sys; print(sys.version)"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"py", "-c", "import sys; print(sys.version)"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"Python", "-c", "print('Python')"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"python3", "-c", "print('Python')"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"py", "-c", "print('Python')"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"Java", "-version"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"Java", "-jar", "myapp.jar"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"Java", "-cp", ".", "MyClass"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"javac", "MyClass.Java"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"javap", "-c", "MyClass"},
+			comm:    "",
+		},
+		{
+			cmdline: []string{"ruby", "prog.rb"},
+			comm:    "ruby",
+		},
+		{
+			cmdline: []string{"puma", "5.6.6", "(tcp://localhost:3000)", "[app]"},
+			comm:    "ruby",
+		},
+		{
+			cmdline: []string{"irb"},
+			comm:    "ruby2.7",
+		},
```

Comment:
Is it worth adding a test case and comment for the jruby case which is being detected as Java for now?

Reply:
Sounds good to me, I'll add that. FYI JRuby will be handled as part of https://datadoghq.atlassian.net/browse/PROC-3233